pkg/reconciler/namespace/namespace.go

```diff
@@ -98,9 +106,10 @@ func (c *Controller) reconcileResource(ctx context.Context, lclusterName string,
 		return nil
 	}
 
-	// Update the resource's assignment.
```

Comment:
nit: revert to 'assignment' from 'scheduled cluster'

Reply:
Reverted.